README.md

```diff
@@ -16,3 +16,8 @@ This plugin offers support to receive and handle native push notifications with
 - [Push notification payload details](docs/PAYLOAD.md)
 - [Contributing](.github/CONTRIBUTING.md)
 - [License (MIT)](MIT-LICENSE)
+
+
+# Do you like tutorial? You get tutorial!
+
+ - [PhoneGap Day EU/US Push Workshop 2016](http://macdonst.github.io/push-workshop/)
```

Comment:
Interesting. I saw both links you posted, and I have them kept "at home" too, for a while now. But upon simple inspection, both use the same repo as a base, and thus I thought they were the same.

If they're both usefull each on it's own, I'll add both then with the info on where they differ.